extensions/amp-story/1.0/test/test-page-advancement.js

```diff
@@ -46,6 +47,31 @@ describes.realWin('page-advancement', {amp: true}, (env) => {
 
         expect(advancement).to.not.be.instanceOf(ManualAdvancement);
       });
+
+      it('should show system UI and unpause on visibilitychange', async () => {
```

Comment:
Nit: plz split this in two so we get one test for the system UI, one test for unpause - it's easier to debug if a test fails